docs/workflow/pycbc_make_coinc_search_workflow.rst

```diff
@@ -940,10 +940,23 @@ If you are running your own build of ``pycbc_inspiral``, you will need to give
 a path to a gsiftp URL and tell Pegasus that the executable is not installed
 on the OSG with the two ``--config-overrides`` options::
 
-    'executables:inspiral:gsiftp://server.hostname/path/to/pycbc_inspiral'
+    "executables:inspiral:gsiftp://server.hostname/path/to/pycbc_inspiral"
+    "pegasus_profile-inspiral:pycbc|installed:False"
```

Comment:
@josh-willis why are you pulling the executable via gridftp, rather than using the bundle that is deployed in CVMFS?  For example:
```
"executables:inspiral:/cvmfs/oasis.opensciencegrid.org/ligo/sw/pycbc/x86_64_rhel_6/bundle/v1.9.4/pycbc_inspiral"
```

Reply:
@duncan-brown Well, that part of the documentation I didn't change (the line you point to). I merely noticed that the text above it referenced two options and gave only the one, so my change was to add the second option.

I assume the original intent was for someone to run a bundled executable that they had built themselves, though I've never personally tested that and don't know whether it still works. But that's what the lines above the changed ones say.

But there are instructions right above those changes which do point the user toward the CVMFS executable; this is just a "secondary" approach, I suppose.